crates/modelardb_compression/src/models/swing.rs

```diff
@@ -326,6 +350,26 @@ fn compute_slope_and_intercept(
     }
 }
 
+/// Computes numerator and denominator for Equation 6 
```

Comment:
Maybe update the doc comments for the function so they describe what the function does like is done for the other functions. The doc comment should be self-contained enough that the reader can understand the purpose of the function, e.g., see `fit_data_pont()`.